app/views/foreman_salt/job_templates/salt_run_state_highstate_-_salt_default.erb

```diff
@@ -0,0 +1,13 @@
+<%#
+name: Salt Run state.highstate - Salt default
+job_category: Salt state
```

Comment:
Can you please change to job category "Salt" because e.g. https://github.com/theforeman/foreman_salt/blob/master/app/views/foreman_salt/job_templates/salt_run_state_highstate_-_ssh_default.erb is already using "Salt"

Reply:
I think the reason I went with this is that originally there was an idea that when running a job you could have multiple providers for a template. It never really took off and the job form looks weird if there are templates with different providers in the same category

Reply:
Understand. Maybe we should change the job category of the other, already existing jobs to ' Salt-Call' and change yours to 'Salt'?